content/blog/esc-software-engineering/index.md

```diff
@@ -0,0 +1,176 @@
+---
+title: "Secrets and Configuration for Software Engineers: Pulumi ESC Versioning, SDKs and More"
+allow_long_title: true
+
+date: 2024-06-05T04:00:00-07:00
+
+meta_desc: "Major new features for Pulumi ESC bring best-in-class software engineering capabilities to secrets and configuration management"
+
+meta_image: meta.png
+
+authors:
+    - luke-hoban
+
+tags:
+    - esc
+    - secrets
+    - platform-engineering
+
+---
+
+Last October we launched [Pulumi ESC](https://www.pulumi.com/docs/esc/), a new secrets and configuration management offering from Pulumi that manages and tames secrets and configuration complexity across all of your cloud applications and infrastructure.  We’ve seen incredible adoption of ESC over the last 6 months, with thousands of developers now managing their secrets and configuration via ESC.
+
+Today, we are excited to announce the most significant collection of Pulumi ESC enhancements since the initial launch, with three major new features that bring best-in-class software engineering capabilities to bear on your Secrets and Configuration management needs.
+
+<!--more-->
+
+* __Versioning__:  Pulumi ESC now supports versioning of environments, allowing you to see and audit every change to the secrets and configuration for an environment, pin references to an environment to a specific version or version tag, and safely roll back an environment to a previous version.
```

Comment:
Do we also want to add links to versioning, SDK, PSP blog posts from here? Similar to how we have in Joe's post: https://www.pulumi.com/blog/infrastructure-lifecycle-management/

Reply:
Yeah - good point - I almost did in my latest edit pass - and you suggesting it as well pushes me over the edge. Will update shortly.